README.md

```diff
@@ -1 +1,54 @@
-# space-travelerers
\ No newline at end of file
+# JFF Space Travelers Hub
+## 📑 Project description
+
+The Space Travelers Hub project was built using ReactJS, SCSS, HTML, and tested through Jest.
+This is a React/Redux project for educational purposes only.
+
```

Comment:
- [ ] [OPTIONAL] Please add a screenshot of your project to your README file. This will be very good for your project documentation